scripts/create_ai_entries.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+import argparse
+import re
+import requests
+import sys
+import time
+
+from typing import List, Optional
+
+from utilities import (
+    logger,
+    build_parser,
+    check_url,
+)
+
+
+TYPE_DICT = {
+    "Binding site": "B",
+    "Conserved site": "C",
+    "Domain": "D",
+    "Family": "F",
+    "Homologous superfamily": "H",
+    "PTM": "P",
+    "Repeat": "R",
+}
+
+
+def main(argv: Optional[List[str]] = None):
+    if argv is None:
+        parser = build_parser()
+        args = parser.parse_args()
+    else:
+        parser = build_parser(argv)
+        args = parser.parse_args()
+
+    check_url(args)
+
+    payload = {
+        "username": args.username,
+        "password": args.password,
+    }
+
+    s = requests.Session()
+    r = s.post(
+        f"{args.pronto}/login/".replace("//login", "/login"),
+        data=payload,
+        timeout=args.timeout,
+    )
+
+    if (s.get(f"{args.pronto}/api/".replace("//api", "/api")).text).find('"user": Null') != -1:
```

Comment:
If the login is successful, Flask creates a client-side cookie, so you can check whether it exists:

```suggestion
    if s.cookies.get("session") is None:
```